examples/js/savedMaps.js

```diff
@@ -7,7 +7,7 @@ saveMapModalBtn.addEventListener('click', () => {
   const savedMaps = JSON.parse(localStorage.getItem('savedMaps'));
   bootstrap.Modal.getInstance(savedMapsModal).show();
 
-  if (savedMaps.length > 0) {
+  if (savedMaps !== null) {
```

Comment:
@jywarren I noticed that when there are no `savedMaps`, the previous check return an error: 
`TypeError: Cannot read properties of null (reading 'length')`
As the `savedMaps` variable would be `null` if there were no maps in the localStorage
And the error prevented the "No Saved Maps" text from displaying (that's how I caught it)
So with this fix, it works fine